screen/ui-article-detail/src/main/java/kasem/sm/ui_detail/DetailVM.kt

```diff
@@ -32,7 +32,7 @@ class DetailVM @Inject constructor(
     savedStateHandle: SavedStateHandle,
 ) : ViewModel() {
 
-    private val articleId = savedStateHandle.get<String>("id")?.toInt() ?: -1
+    private val articleId = savedStateHandle.get<Int>("id") ?: -1
```

Comment:
I remember that I was facing some weird issue here. The app would crash whenever I changed the type to Int. Maybe it was a random issue but I did not want to waste my time and hence haha. Great that it's not working.

Reply:
Yeah, I have also faced that. But using destinations it worked 😊

Reply:
Actually I did a type, "Great that it's now working!" 😃